### PR TITLE
Feature/weather app

### PR DESCRIPTION
**Pull request by Mindo-Joseph**
#### What does this PR do?
- Creates a weather application using vanilla JS through async/await concepts
#### Description of Task to be completed?
- Test your understanding of async/await or Promises by building a weather application
#### How should this be manually tested?
* Clone this repo ```https://github.com/Mindo-Joseph/WeatherApp.git```
* Open terminal```cd``` to the cloned folder
* Checkout to the feature branch by ```git checkout feature/WeatherApp```
* Install dependencies by running ```npm i``` in your terminal
* Run webpack build by using the following command ```npm run build```

### CODE REVIEW

**Review by trekab — COMMENTED** [review]
### Invalid! :stop_sign: 
Hi @Mindo-Joseph! Unfortunately, I have to mark your code review request as invalid because you didn't add a descriptive `README.md` file to your repository as required. Read more from the [Professional GitHub Repository](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) guide in the pathwright and make the necessary changes before your next code review request.

__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack",
+    "start": "webpack --watch"
```
Comment (whiz25) [inline package.json]: - add script for `dev`. you may use something like below.
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray:
You have done a good job so far with the project. Most requirements have been fulfilled but we need to fix a few before we can merge. 
- [ ] fix all linter errors in the PR.
- [ ] check my comment for some suggestions


### Martin Afani

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Good job :+1: This project is approved :white_check_mark:


#### _Reviewed by Temitayo Adeleke_

**Pull request merged**